src/screens/WelcomeScreen.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const landingPage = () => {
```

Comment:
In the future, please do not PR until code is functional and ready for acceptance. This screen needs to be accessible and render in the app, even with placeholder text/images. Also please add functional code to the empty files or remove them from the PR.

Reply:
It's okay to open a PR with unfinished work if it's a `draft` PR :)

Reply:
I agree with, Linden, that if the purpose of this change is to add a screen, that screen should be accessible for validation :) 

Good job so far!